docs/command-line-interface/topaz-cli/configuration.mdx

```diff
@@ -37,6 +39,18 @@ To generate a configuration file for a local OCI image, use the following comman
 topaz configure -d -l my-image:latest -n todo
 ```
 
+## Named configurations
+
+Topaz CLI allows the user to have multiple configuration files and easily switch between them when running the topaz container. 
+
+When using templates the generated configuration file uses the name of the template (ex. for `todo` template it will generate a `$XDG_CONFIG_HOME/topaz/cfg/todo.yaml`).
+
+To see available configuration files you can use the `topaz list` command, this will also highlight the current active configuration file. 
+
+If you want to start a different configuration file, you can just specify the file you want to use in the `topaz start/run --config=<name_of_file.yaml>` command. This will automatically switch you active configuration file to the one you used.
+
+Note: This change affect the topaz container name. This changes from the default `topaz` to the `topaz-<configname>` format. 
```

Comment:
What exactly do you mean by "This change"? Do you mean that the container name is `topaz` by default, but if you run `topaz start -c myfile.yaml` then it will be `topaz-myfile`? Or is it _always_ `topaz-<configname>`?

Reply:
The container will always be in the format `topaz-<configname>` once the named configuration changes are merged.